x-pack/plugin/searchable-snapshots/src/main/java/org/elasticsearch/xpack/searchablesnapshots/cache/blob/CachedBlob.java

```diff
@@ -78,11 +59,13 @@ private CachedBlob(
 
     @Override
     public XContentBuilder toXContent(XContentBuilder builder, Params params) throws IOException {
+        @UpdateForV9    // we can remove the version field when we no longer need to keep compatibility with <8.12
+        final int version812 = 8_12_00_99;  // hard-coded to 8.12
```

Comment:
Should it be changed when we release 8.13 as well?

Reply:
This field is only needed for bwc because previous versions blow up without the `version` field. If it doesn't need to be any particular value, then we might as well just hard-code it

Reply:
We could also leave it as `Version.CURRENT.id`, and then remove it entirely in v9, what do you think?

Reply:
I am not sure how it is used. I was only surprised by hard-coded value, wanted to confirm that was intentional.

Reply:
It was, but I've re-thought it, and changed to `Version.CURRENT.id`

Reply:
Thanks for spotting this @idegtiarenko. It makes sense to follow the current version; if I remember correctly this field is used to know if the bytes stored in the snapshot blob cache index can be read by the node - otherwise they are discarded.

Reply:
This field was not actually referenced anywhere, hence why it's removed here...